<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:context="http://www.springframework.org/schema/context"
       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/context https://www.springframework.org/schema/context/spring-context.xsd">

    <!-- primary="true" work same @Primary for fix error have the same beans -->

    <bean name="dataSourceMySQL" id="dataSourceMySQL" class="org.springframework.jdbc.datasource.DriverManagerDataSource" > <!--destroy-method="close"-->
        <property name="driverClassName" value="com.mysql.cj.jdbc.Driver"/>
        <property name="url" value="jdbc:mysql://localhost:3306/ttknp"/>
        <property name="username" value="root"/>
        <property name="password" value=""/>
    </bean>

</beans>